include/rocksdb/write_buffer_manager.h

```diff
@@ -81,13 +81,20 @@ class WriteBufferManager final {
     return buffer_size_.load(std::memory_order_relaxed);
   }
 
+  // REQUIRED: `new_size` > 0
   void SetBufferSize(size_t new_size) {
+    assert(new_size > 0);
```

Comment:
Misc: mainly for translating the removed comment `// Cannot early-exit on !enabled() because SetBufferSize(0) needs to unblock the writers.` in `WriteBufferManager::MaybeEndWriteStall()` in code